GlobalVariables.js

```diff
@@ -19,6 +19,9 @@ class GlobalVariables {
         this.lastOrderWasFilled = true;
         this.fills = 0;
         this.tradeHistory= null;
+        this.lastMarketRatio = 0; //buyers/sellers
+        this.currentMarketRatio = 0; //buyers/sellers
```

Comment:
i `...Ratio` sono stati deprecati perché "splittati" in `buySpeed` e `sellSpeed`